gzip_cache/gzip_cache.py

```diff
@@ -65,19 +65,23 @@ def create_gzip_cache(pelican):
     '''
     for dirpath, _, filenames in os.walk(pelican.settings['OUTPUT_PATH']):
         for name in filenames:
-            if should_compress(name):
+            if should_compress(name, pelican.settings):
                 filepath = os.path.join(dirpath, name)
                 create_gzip_file(filepath, should_overwrite(pelican.settings))
 
 
-def should_compress(filename):
+def should_compress(filename, settings):
     '''Check if the filename is a type of file that should be compressed.
 
     :param filename: A file name to check against
+    :param settings: The pelican instance settings
     '''
     for extension in EXCLUDE_TYPES:
         if filename.endswith(extension):
             return False
+    for extension in settings.get('GZIP_CACHE_EXCLUDE_TYPES', {}):
```

Comment:
Setting is a `list`, so why is the fallback value a `dict`?

Reply:
good point

Reply:
Done